### ISSUE TITLE
support @opentofu(required_version)

### ISSUE THREAD

**Issue by kvendingoldo** [tofuutils/tenv#179]
**Is your feature request related to a problem? Please describe.**
We have to handle the case described in https://github.com/opentofu/opentofu/issues/1005 after the final discussion from OpenTofu maintainers side.

**Comment by dvaumoron** [comment]
We should think to handle `.tofu` files as well

**Comment by kvendingoldo** [comment]
You're right, but we need to wait until the discussion in opentofu community will be completed and converted to a standard.

**Comment by dvaumoron** [comment]
`.tofu` file are in 1.8 alpha, i will do the change soon in order to be ready

**Comment by kvendingoldo** [comment]
Yep, wanted to highlight it either :)

**Comment by kvendingoldo** [comment]
I think that we need to keep it open, until the [RFC](https://github.com/opentofu/opentofu/pull/1716) won't be finished

**Comment by kvendingoldo** [comment]
> Hi everyone!

This feature is now implemented enough for the v1.10 release. We're deferring https://github.com/opentofu/opentofu/issues/2751 for later work, but since that's already tracked as a separate issue I'm going to close this one to represent that the main work is completed.

Thanks!
